pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/model/patch/OncHistoryDetailPatchTest.java

```diff
@@ -131,12 +135,75 @@ public void doPatchTest() {
             Map<String, String> patchMap = nameValuesToMap(patch.getNameValues());
             Assert.assertEquals(patchMap.get("oD.destructionPolicy"), oncHistoryDetail.getDestructionPolicy());
             Assert.assertEquals(patchMap.get("oD.facility"), oncHistoryDetail.getFacility());
+
+            changeFaxSent(oncHistoryDetail.getOncHistoryDetailId(), participantId);
+
         } catch (Exception e) {
             e.printStackTrace();
             Assert.fail("Unexpected exception: " + e);
         }
     }
 
+    public void changeFaxSent(int oncHistoryDetailId, int participantId) throws Exception {
+        String patchJson = TestUtil.readFile("onchistory/faxSentPatch.json");
+        Gson gson = new Gson();
+        Patch patch = gson.fromJson(patchJson, Patch.class);
+        patch.setDdpParticipantId(ddpParticipantId);
+        patch.setParentId(Integer.toString(participantId));
+        patch.setRealm(instanceName);
+        patch.setId(String.valueOf(oncHistoryDetailId));
+
+        NotificationUtil mockNotificationUtil = mock(NotificationUtil.class);
+        BasePatch existingOncHistoryDetailPatch = ExistingRecordPatchFactory.produce(patch, mockNotificationUtil);
+        existingOncHistoryDetailPatch.setElasticSearchExportable(true);
+        existingOncHistoryDetailPatch.doPatch();
+
+        try {
+            TimeUnit.SECONDS.sleep(1);
```

Comment:
What are we waiting for here?  ES sync?

Reply:
Yes, found this in the other tests in this file too, and if I remove it sometimes ES does not catch up in time